cms/djangoapps/contentstore/views/preview.py

```diff
@@ -139,6 +140,17 @@ def _preview_module_system(request, descriptor):
 
     descriptor.runtime._services['user'] = StudioUserService(request)  # pylint: disable=protected-access
 
+    services_list = {
```

Comment:
By calling this param a 'list', I would expect to see an array/list.  Maybe just call it 'services', or 'services_data'?